main.js

```diff
@@ -33,7 +33,10 @@ function setup() {
 
 	// Set the frame rate
 	frameRate(fr)
-
+  targetScoreInput = createInput('')
```

Comment:
The target score should have a default value in the input so that you user is not forced to pick one before starting.  (Probably in a different issue/pr) change this to a slider bar to eliminate the validation check.